cluster/sbin/snappy-nodes.sh

```diff
@@ -448,4 +451,18 @@ else
   fi
   execute "$@"
 fi
-wait
+
+if [ "$isServerStart" ]; then
+  # server status file
+  SERVERS_STATUS_FILE="$SNAPPY_HOME/work/serversstatus"
```

Comment:
There is already a members.txt, so better name would be members-status.txt

Reply:
Ok.